bun/PKGBUILD

```
@@ -1,17 +1,23 @@
 # Maintainer: Axel Navarro <navarroaxel gmail>
 pkgname=bun
 pkgver=0.1.5
-pkgrel=1
+pkgrel=2
 pkgdesc="A runtime for JavaScript and TypeScript"
-arch=(x86_64)
+arch=('x86_64' 'aarch64')
 url="https://github.com/Jarred-Sumner/bun-releases-for-updater"
 license=(MIT)
 provides=(bun)
-source=(${pkgname}-${pkgver}.zip::$url/releases/download/$pkgname-v$pkgver/$pkgname-linux-x64.zip)
-sha256sums=('524a2d7e51ddda4786799552ae1c18ab8e6173bb30f158f26cae899a7e74f22f')
+source_x86_64=(${pkgname}-${pkgver}.zip::$url/releases/download/$pkgname-v$pkgver/$pkgname-linux-x64.zip)
+sha256sums_x86_64=('524a2d7e51ddda4786799552ae1c18ab8e6173bb30f158f26cae899a7e74f22f')
+source_aarch64=(${pkgname}-${pkgver}.zip::$url/releases/download/$pkgname-v$pkgver/$pkgname-linux-aarch64.zip)
```

All filenames should be different to allow the usage of `makepkg --allsource` for testing.
```suggestion
source_aarch64=(${pkgname}-${pkgver}-aarch64.zip::$url/releases/download/$pkgname-v$pkgver/$pkgname-linux-aarch64.zip)
```

---

Good to know. First time in a PKGBUILD ^^;